src/fides/api/ops/schemas/redis_cache.py

```diff
@@ -11,6 +11,7 @@ class Identity(BaseSchema):
 
     phone_number: Optional[str] = None
     email: Optional[str] = None
+    user_id: Optional[str] = None
```

Comment:
Have we confirmed this is the user_id that's needed here?  https://support.google.com/analytics/answer/3123669?hl=en, I just assumed that in the design docs, but I don't know for sure